projects/novo-elements/src/elements/select/Select.ts

```diff
@@ -246,7 +246,7 @@ export class NovoSelectElement
   @Input() displayWith: ((value: any) => string) | null = null;
   /** * Function to compare the option values with the selected values. */
   @Input() compareWith: (o1: any, o2: any) => boolean = (o1: any, o2: any) =>
-    o1 === o2 || o1 === o2.id || (!Helpers.isEmpty(o1.id) && !Helpers.isEmpty(o2.id) && o1.id === o2.id);
+    o1 === o2 || o1 === o2?.id || (!Helpers.isEmpty(o1?.id) && !Helpers.isEmpty(o2?.id) && o1.id === o2.id);
```

Comment:
Would this still return correct values if you compare `o1 = undefined` with `o2 = { name: 'Bob' }`? I'm worried that `o2?.id` will evaluate to undefined, and that's equal to o1. Could be worth some added unit tests around. I feel like that's the general goal of the Helpers.isEmpty functions.

Reply:
This was a quick add-on, addressing a console error. Can remove if you think not worth adding here.
However, wouldn't the original `o1 === o2.id` already allow for that _undefined_ scenario?